how-we-work.md

```diff
@@ -0,0 +1 @@
+# How We Wrok
```

Comment:
There's a typo in the heading, hopefully the reviewer catches it and blocks requests a change.